src/components/Product/ProductList.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+
+import appStyles from '@/App.module.css';
+import FiltersComponent from '@/components/Filters/Filters.component.tsx';
+import Pagination from '@/components/Pagination/Pagination.component.tsx';
+import styles from '@/components/Product/Product.module.css';
+import ProductCard from '@/components/Product/ProductCard.tsx';
+import type { Category } from '@/constants/filtersCategory.ts';
+import { SortOption } from '@/constants/filtersSortOption.ts';
+import mockData from '@/Models/mocData.json';
+import type { Product } from '@/Models/product.ts';
+
+const ProductList: React.FC = () => {
```

Comment:
<img width="1352" alt="Screenshot 2024-06-23 at 12 44 49" src="https://github.com/igor0806/fe-react-2024/assets/102523692/319a820b-f928-40f0-9987-c6a91869cffa">

Products should be placed near each other